### PR TITLE
Exit with non-zero error code when passed an invalid command or an invalid parameter.

### PR DESCRIPTION
**Pull request by sersut**
/cc: @chef/engineering-services 

Fixes https://github.com/chef/chef-acceptance/issues/41

### CODE REVIEW

**Comment by scotthain** [comment]
👍

File: spec/integration/cli_spec.rb [hunk 1]
```diff
@@ -31,6 +31,10 @@ def run_acceptance
     end
   end
 
+  it "exits with non-zero error code", :focus do
```
Comment (schisamo) [inline spec/integration/cli_spec.rb]: A stray `:focus` is coming along for a ride.

**Comment by schisamo** [comment]
@sersut Can we please add some test coverage that passing an invalid CLI option causes a non-zero exit?

**Comment by sersut** [comment]
@schisamo Will do!

File: spec/integration/cli_spec.rb (hunk 1 above)
Comment (sersut) [inline spec/integration/cli_spec.rb]: :facepalm:
